packages/openapi-v3/src/__tests__/unit/decorators/request-body/request-body-shortcut.decorator.unit.ts

```diff
@@ -33,10 +30,41 @@ describe('requestBody decorator - shortcuts', () => {
 
       const requestBodySpec = actualSpec.paths['/greeting']['post'].requestBody;
       expect(requestBodySpec).to.have.properties({
-        description: description,
+        description,
         required: false,
         content: expectedContent,
       });
     });
   });
+
+  context('file', () => {
+    it('generates the correct schema spec for an file argument', () => {
+      const description = 'a picture';
+      class MyController {
+        @post('/pictures')
+        upload(
+          @requestBody.file({description, required: true})
+          request: unknown, // It should be `Request` from `@loopback/rest`
```

Comment:
hmm any reason cannot use `Request`?

Reply:
We don't want to add `@loopback/rest` as dependency to import `Request`.